cmd/bootstrapper/initialize_kubelet.go

```diff
@@ -65,7 +65,8 @@ func runInitializeKubeletCmd(cmd *cobra.Command, args []string) {
 		log.Error(err, "could not run bootstrapper")
 		os.Exit(1)
 	} else {
-		log.Info("Bootstrapping completed successfully")
+		// Send success message to StdOut for WSU to ascertain that bootstrapping was successful
+		os.Stdout.WriteString("Bootstrapping completed successfully")
```

Comment:
Didn't we agree to do the opposite. Log to stderr explicitly and logger should be configured to write to stdout?